crates/iota-sdk-ffi/src/crypto/zklogin.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use iota_crypto::Verifier;
+use iota_types::{Jwk, JwkId};
+
+use crate::{error::Result, types::crypto::zklogin::ZkLoginAuthenticator};
+
+#[derive(derive_more::From, uniffi::Object)]
+pub struct ZkloginVerifier(pub iota_crypto::zklogin::ZkloginVerifier);
+
+#[uniffi::export]
+impl ZkloginVerifier {
+    #[uniffi::constructor]
+    pub fn new_mainnet() -> Self {
+        Self(iota_crypto::zklogin::ZkloginVerifier::new_mainnet())
+    }
+
+    #[uniffi::constructor]
+    pub fn new_dev() -> Self {
```

Comment:
I know the comment is also not in crates/iota-crypto/src/zklogin/mod.rs, but would be good to have it there and here
```
/// Load a fixed verifying key from zkLogin.vkey output. This is based on a
/// local setup and should not use in production.
```